build-server/src/main/resources/dockerfiles/java-maven/Dockerfile

```diff
@@ -36,11 +35,17 @@ RUN echo 'exit $exit_value' >> /usr/bin/with-xvfb
 RUN chmod +x /usr/bin/with-xvfb
 
 RUN echo '#!/bin/bash' >> /usr/bin/fetch-deps
-RUN echo 'git clone https://github.com/devhub-tud/ti2206-template.git' >> /usr/bin/fetch-deps
-RUN echo 'cd ti2206-template' >> /usr/bin/fetch-deps
-RUN echo 'with-xvfb mvn -B clean package' >> /usr/bin/fetch-deps
+RUN echo 'git clone https://github.com/SERG-Delft/jpacman-framework.git' >> /usr/bin/fetch-deps
+RUN echo 'cd jpacman-framework' >> /usr/bin/fetch-deps
+RUN echo 'git checkout 5f132d16d9258026ab1a19a973c23f275c1ad0e9' >> /usr/bin/fetch-deps
+RUN echo 'with-xvfb mvn -B clean install' >> /usr/bin/fetch-deps
```

Comment:
Install `jpacman-framework` (as the repository is down). Version 6.0.0 is checked out.